### PR TITLE
Cleanup Ibft executors

### PR DESCRIPTION
**Pull request by rain-on**
Explicitly clean up executors used for IBFT timers.

### CODE REVIEW

File: pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java [hunk 1]
```diff
@@ -214,21 +215,20 @@
 
     final IbftGossip gossiper = new IbftGossip(uniqueMessageMulticaster);
 
+    final ScheduledExecutorService timerExecutor = Executors.newSingleThreadScheduledExecutor();
```
Comment (jframe) [inline pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java]: Could be good idea to use the MonitoredExecutors.newFixedThreadPool so we can give our thread pool a name and get some free metrics as well.

File: pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java [hunk 2]
```diff
@@ -286,6 +286,12 @@
           } catch (final InterruptedException e) {
             LOG.error("Failed to shutdown ibft processor executor");
           }
+          timerExecutor.shutdownNow();
+          try {
+            timerExecutor.awaitTermination(5, TimeUnit.SECONDS);
+          } catch (final InterruptedException e) {
+            LOG.error("Failed to shutdown timerexecutor");
```
Comment (jframe) [inline pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java]: nit: thinking this should be "Failed to shutdown timer executor"

Comment (rain-on) [reply pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java]: done.

File: pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java (hunk 1 above)
Comment (rain-on) [reply pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftPantheonController.java]: done.

**Pull request merged**